pkgs/top-level/python-packages.nix

```diff
@@ -5906,6 +5906,8 @@ in {
     });
   in if isPy3k then pyxattr' else pyxattr_2;
 
+  pyxnat = callPackage ../development/python-modules/pyxnat { };
+
   pyx = callPackage ../development/python-modules/pyx { };
 
   pyxdg = callPackage ../development/python-modules/pyxdg { };
```

Comment:
this should be lower, at least after `pyxdg`

Reply:
Is it OK if I move `pyx` slightly?

Reply:
Never mind, moved.